carma-messenger-ui/website/widgets/eventManagement/widget.js

```diff
@@ -112,23 +112,32 @@ CarmaJS.WidgetFramework.eventManagement = (function () {
         var DownTrackValue =  $('#DownTrack').val();
         var MinGapValue =  $('#MinGap').val();
         var AdvisorySpeedValue =  $('#AdvisorySpeed').val();
-        if(UpTrackValue==""){
-            alert("Up Track value is required and should be greater than 0.");
+        if(UpTrackValue=="" || isNaN(UpTrackValue) 
+            || UpTrackValue > CarmaJS.Config.getUpTrackRange().MAX 
+            || UpTrackValue < CarmaJS.Config.getUpTrackRange().MIN)
+        {
+            alert("Up Track value is required, should be greater than "+CarmaJS.Config.getUpTrackRange().MIN+" and less than "+ CarmaJS.Config.getUpTrackRange().MAX);
```

Comment:
this say should be "greater than or equal to", and "less than or equal to". I think your IF statemements are correct, just this alert message needs to be adjusted. This applies for all below as well.